recipe-server/client/control/tests/state/experimenter/test_actions.js

```diff
@@ -0,0 +1,169 @@
+import { fromJS } from 'immutable';
+import * as matchers from 'jasmine-immutable-matchers';
+import fetchMock from 'fetch-mock';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+
+import {
+  acceptExperiment,
+  fetchExperiments,
+  getPendingExperimentsUrl,
+  importExperiment,
+  rejectExperiment,
+  selectRejectedExperiment,
+} from 'control/state/app/experimenter/actions';
+
+import ExperimentFactory, {
+  createRecipeResponse,
+} from 'control/tests/state/experimenter';
+
+import {
+  EXPERIMENTS_RECEIVE,
+  EXPERIMENT_ACCEPTED,
+  EXPERIMENT_REJECTED,
+  EXPERIMENT_SELECT_REJECTED,
+  RECIPE_RECEIVE,
+} from 'control/state/action-types';
+
+const middlewares = [thunk];
+const mockStore = configureStore(middlewares);
+
+describe('Experimenter Actions fetchExperiments', () => {
+  beforeEach(() => {
+    jasmine.addMatchers(matchers);
```

Comment:
I imagine this could be done once at top level, instead of `beforeEach`. I think that would make the tests very slightly faster, and simpler.